tests/test_fetch_topic.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import requests
+import json
+
+from random import randrange
+
+from councilmatic_core.models import Bill, Subject
+from lametro.models import SubjectGuid
+from lametro.views import fetch_topic
+
+def test_fetch_single_topic(client, subject_guid):
+
+    '''
+    This tests returning a single Subject matching the GUID
+    '''
+
+    s_g = subject_guid.build()
+
+    response = client.get('/topic/', { 'guid': s_g.guid })
+    response = json.loads(response.content.decode('utf-8'))
+    assert response['status_code'] == 200
```

Comment:
how about also checking that the subject that came back is the subject from your subject guid?